crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -109,6 +109,8 @@ where
 
     let (piece_cache, piece_cache_worker) = PieceCache::new(node_client.clone(), peer_id);
 
+    let metrics_endpoints_are_specified = !metrics_endpoints.is_empty();
+
     let (node, mut node_runner, metrics_registry) = {
```

Comment:
Registry shouldn't be created in `configure_dsn` either, it should be created beforehand and we'll use it for not just DSN in the future. `configure_dsn` should take registry itself as an agrument.

Reply:
I'll move it when we have another usage for sure.